debian/control.in

```diff
@@ -23,21 +23,21 @@ Build-Depends: appstream-util,
                libwebkit2gtk-4.0-dev (>= 2.19.2),
                libsqlite3-dev,
                yelp-xsl (>= 3.27.1)
-Standards-Version: 4.3.0
+Standards-Version: 4.5.0
 Vcs-Browser: https://salsa.debian.org/gnome-team/yelp
 Vcs-Git: https://salsa.debian.org/gnome-team/yelp.git
 Homepage: https://wiki.gnome.org/Apps/Yelp
 
 Package: yelp
 Architecture: any
-XCBS-EOS-AppId: org.gnome.Yelp
+XCBS-EOS-AppId: org.gnome.Yelp,
```

Comment:
Is the eol "comma" needed here? Would be good to keep it in sync with `control`.

Reply:
Oops, yes, fixed.